icu4c/source/test/intltest/numbertest_api.cpp

```diff
@@ -753,21 +753,17 @@ void NumberFormatterApiTest::unitMeasure() {
             4.28571,
             u"4 metric tons, 285 kilograms, 710 grams");
 
-//     // TODO(icu-units#73): deal with this "1 foot 12 inches" problem.
-//     // At the time of writing, this test would pass, but is commented out
-//     // because it reflects undesired behaviour:
-//     assertFormatSingle(
-//             u"Demonstrating the \"1 foot 12 inches\" problem",
-//             nullptr,
-//             u"unit/foot-and-inch",
-//             NumberFormatter::with()
-//                 .unit(MeasureUnit::forIdentifier("foot-and-inch", status))
-//                 .precision(Precision::maxSignificantDigits(4))
-//                 .unitWidth(UNUM_UNIT_WIDTH_FULL_NAME),
-//             Locale("en-US"),
-//             1.9999,
-//             // This is undesireable but current behaviour:
-//             u"1 foot, 12 inches");
+    assertFormatSingle(
+            u"Testing  \"1 foot 12 inches\"",
+            nullptr,
+            u"unit/foot-and-inch",
+            NumberFormatter::with()
+                .unit(MeasureUnit::forIdentifier("foot-and-inch", status))
+                .precision(Precision::maxSignificantDigits(4))
+                .unitWidth(UNUM_UNIT_WIDTH_FULL_NAME),
+            Locale("en-US"),
+            1.9999,
+            u"2 feet, 0 inches");
```

Comment:
To consider for the future: might we want to get smarter and return only "2 feet"? (This would mean some loss of end-user insight into the precision of the measurement.)

Reply:
We concluded this could be some additional display option, but on its own, "2 feet, 0 inches" is correct and desireable output.

Reply:
I filed icu-units#101 for this. Closing.